File: mathgame.py
import random #variables and imports
x = 0
y = 0
solution = 0
continuegame = True
correct = 0
mathmode = 0
clientinput = 0

def serverlogic(): #this function creates the logic for the question and solution
    global x, y, solution, mathmode
    mathmode = random.randint(1, 3)
    x = random.randint(1, 10)
    y = random.randint(1, 10)
    if mathmode == 1:
        solution = x + y
        print("What is " + str(x) + " + " + str(y) + "?")
    elif mathmode == 2:
        solution = x - y
        print("What is " + str(x) + " - " + str(y) + "?")
    elif mathmode == 3:
        solution = x * y
        print("What is " + str(x) + " x " + str(y) + "?")
    else:
        print("???")

def clientlogic(): #this fuction collects the user data and processes it
    global clientinput, solution, correct, continuegame
    clientinput = int(input())
    if clientinput == solution:
        print("Correct!")
        correct = correct + 1
    else:
        print("WRONG")
        continuegame = False

while continuegame == True: #loop
    serverlogic()
    clientlogic()

print("You got " + str(correct) + " right!") #end

File: Java stuff/MathGame.java
import java.util.*;

public class MathGame {
    static int x = 0;
    static int y = 0;
    static int solution = 0;
    static boolean continuegame = true;
    static int correct = 0;
    static int mathmode = 0;
    static int clientinput = 0;
    public static void serverlogic() {
        Random rand = new Random();
        mathmode = rand.nextInt(3) + 1;
        x = rand.nextInt(10) + 1;
        y = rand.nextInt(10) + 1;
        if (mathmode == 1) {
            solution = x + y;
            System.out.println("What is " + x + " + " + y + "?");
        } else if (mathmode == 2) {
            solution = x - y;
            System.out.println("What is " + x + " - " + y + "?");
        } else if (mathmode == 3) {
            solution = x * y;
            System.out.println("What is " + x + " x " + y + "?");
        } else {
            System.out.println("???");
        }
    }
    public static void clientlogic() {
        Scanner scanner = new Scanner(System.in);
        clientinput = scanner.nextInt();
        if (clientinput == solution) {
            System.out.println("Correct!");
            correct = correct + 1;
        } else {
            System.out.println("Wrong! End of streak...");
            continuegame = false;
        }
    }
    public static void main(String[] args) {
        while (continuegame == true) {
            serverlogic();
            clientlogic();
        }
        System.out.println("You did a streak of " + correct + " correct answers! Congratulations!");
    }
}